lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1210,6 +1211,18 @@ tryParseIntelFPGAAnnotationString(StringRef AnnotatedCode) {
   return Decorates;
 }
 
+std::vector<SPIRVWord> getBankBitsFromString(StringRef S) {
```

Comment:
Probably in the future we should make this function more generic(add a separator arg) and move it to OCLUtil.